api-docs/api-reference/methods/users/get-accessible-domains-for-user-v2.0.rst

```diff
@@ -0,0 +1,93 @@
+.. _get-accessible-domains-for-user-v2.0:
+
+Get accessible domains for user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/{userId}/RAX-AUTH/domains
+
+Use the get accessible domains for user operation to retrieve the list of
+domains a user has access to.
+
+Request
+-------
+
+This table shows the header and URI parameters for the request:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    X-Auth-Token, Header String *(Required)*, A valid authentication token.
+    {userId}, URI String *(Required)*, A user ID assigned by system when user is added.
+
+Response
+--------
+
+The following table shows the body parameters for the retrieve domains
+response:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    RAX-AUTH:domains, Object *(Required)*, The collection of domains that the authenticated user has permission to view.
+    RAX-AUTH:domains.\ **RAX-AUTH:domain**,Object *(Required)*, An object that contains the domain configuration attribute settings.
+    RAX-AUTH:domain.\ **id**, String *(Required)*, The unique id for the domain.
+    RAX-AUTH:domain.\ **sessionInactivityTimeout**, String *(Required)*, Session inactivity timeout property used across all Rackspace UIs.
```

Comment:
In this case, I was just following what was done before. I think we should always add the namespace of the object from non-core objects.